src/Tools/_framework/ToolPanels/Gradebook.jsx

```diff
@@ -715,7 +722,12 @@ function GradebookOverview() {
           {
             Header: (
               <div>
-                {`${category} Total`} {description}{' '}
+                {`${category} Total`}{' '}
+                {description.length && (
```

Comment:
This tacks a "0" at the end of categories with no description.

Reply:
Also, I think you can delete description on line 743 when there are no assignments.